src/lib/components/Logo/Logo.svelte

```diff
@@ -0,0 +1,146 @@
+  import stackedLogoAvif2x from "$lib/assets/ldaf-stacked-white-rgb-1200px.png?format=avif&quality=85&imagetools";
+
+  import horizLogo from "$lib/assets/ldaf-horiz-white-rgb-600px.png?quality=85&imagetools";
+  import {
+    width as horizLogoWidth,
+    height as horizLogoHeight,
+  } from "$lib/assets/ldaf-horiz-white-rgb-600px.png?blurhash";
+  import horizLogo2x from "$lib/assets/ldaf-horiz-white-rgb-1200px.png?quality=85&imagetools";
+  import horizLogoWebp from "$lib/assets/ldaf-horiz-white-rgb-600px.png?format=webp&quality=85&imagetools";
+  import horizLogoWebp2x from "$lib/assets/ldaf-horiz-white-rgb-1200px.png?format=webp&quality=85&imagetools";
+  import horizLogoAvif from "$lib/assets/ldaf-horiz-white-rgb-600px.png?format=avif&quality=85&imagetools";
+  import horizLogoAvif2x from "$lib/assets/ldaf-horiz-white-rgb-1200px.png?format=avif&quality=85&imagetools";
+
+  import Image from "$lib/components/Image";
+  import type { Placement } from "./logoOptions";
+
+  const logoMap: Record<
+    Placement,
+    {
+      logo: string;
+      logo2x: string;
+      width: number;
+      height: number;
+      logoWebp: string;
+      logoWebp2x: string;
+      logoAvif: string;
+      logoAvif2x: string;
+    }
+  > = {
+    "mobile-header-main": {
+      logo: textLogo,
+      logo2x: textLogo2x,
+      width: textLogoWidth,
+      height: textLogoHeight,
+      logoWebp: textLogoWebp,
+      logoWebp2x: textLogoWebp2x,
+      logoAvif: textLogoAvif,
+      logoAvif2x: textLogoAvif2x,
+    },
+    "mobile-header-menu": {
+      logo: stackedCommissionerLogo,
+      logo2x: stackedCommissionerLogo2x,
+      width: stackedCommissionerLogoWidth,
+      height: stackedCommissionerLogoHeight,
+      logoWebp: stackedCommissionerLogoWebp,
+      logoWebp2x: stackedCommissionerLogoWebp2x,
+      logoAvif: stackedCommissionerLogoAvif,
+      logoAvif2x: stackedCommissionerLogoAvif2x,
+    },
+    "desktop-header": {
+      logo: horizWideCommissionerLogo,
+      logo2x: horizWideCommissionerLogo2x,
+      width: horizWideCommissionerLogoWidth,
+      height: horizWideCommissionerLogoHeight,
+      logoWebp: horizWideCommissionerLogoWebp,
+      logoWebp2x: horizWideCommissionerLogoWebp2x,
+      logoAvif: horizWideCommissionerLogoAvif,
+      logoAvif2x: horizWideCommissionerLogoAvif2x,
+    },
+    "mobile-footer": {
+      logo: stackedLogo,
+      logo2x: stackedLogo2x,
+      width: stackedLogoWidth,
+      height: stackedLogoHeight,
+      logoWebp: stackedLogoWebp,
+      logoWebp2x: stackedLogoWebp2x,
+      logoAvif: stackedLogoAvif,
+      logoAvif2x: stackedLogoAvif2x,
+    },
+    "desktop-footer": {
+      logo: horizLogo,
+      logo2x: horizLogo2x,
+      width: horizLogoWidth,
+      height: horizLogoHeight,
+      logoWebp: horizLogoWebp,
+      logoWebp2x: horizLogoWebp2x,
+      logoAvif: horizLogoAvif,
+      logoAvif2x: horizLogoAvif2x,
+    },
+  };
```

Comment:
This whole block feels pretty dirty, but the alternative was to try to set up dynamic imports, which felt like overkill.

Reply:
Yeah, I mean, as long as the `Image` component is doing the heavy lifting on this and we're _loading_ it performantly, I think we're in the clear